scarb/src/bin/scarb/args.rs

```diff
@@ -134,6 +136,8 @@ pub enum Command {
     Cache(CacheSubcommand),
     /// Remove generated artifacts.
     Clean,
+    /// Print cache path.
+    CachePath,
```

Comment:
Hey! `scarb cache clean` seems enough, I do not think we need an alias for that here.

Reply:
So we don't need a `scarb cache-path` cli command?